src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_help.py

```diff
@@ -53,3 +53,42 @@
     type: command
     short-summary: Show the price sheet for Azure subscription within a current billing period or specified billing period
 """
+
+helps['consumption marketplace'] = """
+    type: group
+    short-summary: Inspect the marketplace by Azure subscription within current or specified billing period.
+"""
+
+helps['consumption marketplace list'] = """
+    type: command
+    short-summary: Show the marketplace for Azure subscription within a current billing period or specified billing period
+"""
+
+helps['consumption budget'] = """
+    type: group
+    short-summary: Offers ability to create, get, update, and delete budgets for Azure subscription optionally by resource group name
+"""
+
+helps['consumption budget list'] = """
+    type: command
+    short-summary: Offers ability to create, get, update, and delete budgets for Azure subscription optionally by resource group name
```

Comment:
I have doubts that your list command has all these capabilities.

Reply:
@tjprescott -Thanks for the comments. I've addressed all but one. Where did you notice 'overriding the global aliasing of resource group' ?